frontend/src/components/table/Table.js

```diff
@@ -174,14 +187,12 @@ export default class Table extends PureComponent {
     }
 
     switch (e.key) {
-      case 'ArrowDown': {
+      case ARROW_DOWN_KEY: {
         e.preventDefault();
 
         const { currentId, array } = this.getCurrentRowId();
```

Comment:
provide the proper param. don't assume default param.

Reply:
ok fixing to 

const { currentId, array } = this.getCurrentRowId(ARROW_DOWN_KEY);